components/suggest/src/store.rs

```diff
@@ -157,6 +174,34 @@ pub struct SuggestIngestionConstraints {
     pub max_suggestions: Option<u64>,
 }
 
+/// Global Suggest configuration data.
```

Comment:
Small suggestion: Shall we put these types in a new `config.rs` module?

Reply:
Sounds good!